GradedModulePresentationsForCAP/gap/GradedModulePresentationMorphism.gi

```diff
@@ -320,3 +320,20 @@ end );
 ##
 ####################################
 
+##
+InstallMethod( Display,
+               [ IsGradedLeftOrRightPresentationMorphism ],
+               # FIXME: Fix the rank in GenericView and delete this afterwards
+               9999,
+               
+  function( morphism )
+    
+    Display( UnderlyingMatrix( morphism ) );
+    
+    Print( "\n" );
+    
+    Print( StringMutable( morphism ) );
```

Comment:
Can you use String instead? It is not important, but `String` is an attribute, whereas `StringMutable` copies the string (which is not necessary here).